E3Next/Processors/DebuffDot.cs

```diff
@@ -70,8 +70,9 @@ public static void Check_OffAssistSpells()
             //do not off assist if you are in the middle of gather dusk. It sucks to put it on an add. 
             bool duskfall = MQ.Query<bool>("${$Bool[${Me.Song[Fading Light]}]}");
             if (duskfall) return;
-            duskfall = MQ.Query<bool>("${$Bool[${Me.Song[Gathering Dusk]}]}");
-            if (duskfall) return;
+            int gatheringDuskTicks = MQ.Query<int>("${Me.Song[Gathering Dusk].Duration.Ticks}");
+
+            if (gatheringDuskTicks <=2) return;
```

Comment:
and if you don't have gathering dusk, it returns -1, and returns? :) so basically its always disabled?